osCam/templates/core/navigation.html

```diff
@@ -1,14 +1,16 @@
 <nav class="navbar navbar-expand-md navbar-light " style="background-color: grey;">
     <a class="navbar-brand " href="/">Home </a>
       <ul class="navbar-nav mr-auto ">
-          {% if user.is_authenticated %}
+         {% if not user.is_authenticated %}
           <li>
-            <a class="nav-link" href="/logout/">Logout</a>
+            <a class="nav-link" href="/login/">Login  </a>
           </li>
-          {% else %}
+          {% endif %}
+          {% if user.is_authenticated %}
           <li>
-            <a class="nav-link" href="/login/">Login  </a>
+            <a class="nav-link" href="/config/">Config  </a>
```

Comment:
looks good

Reply:
i thought config sounded cooler than settings but this could be a good survey question if we are doing any more surveys.